src/Azure.Functions.Cli/Helpers/TargetFrameworkHelper.cs

```diff
@@ -5,11 +5,17 @@ namespace Azure.Functions.Cli.Helpers
 {
     public static class TargetFrameworkHelper
     {
-        private static IEnumerable<string> supportedTargetFrameworks = new string[] { TargetFramework.net48, TargetFramework.net6, TargetFramework.net7, TargetFramework.net8 };
+        private static readonly IEnumerable<string> supportedTargetFrameworks = new string[] { TargetFramework.net48, TargetFramework.net6, TargetFramework.net7, TargetFramework.net8 };
+        private static readonly IEnumerable<string> supportedInProcTargetFrameworks = new string[] { TargetFramework.net6, TargetFramework.net8 };
```

Comment:
Since this is used in string construction (see other comment), can we reverse the order of these so that we lead with .NET 8 in both cases?

Reply:
Providing an actual suggestion for convenience:
```suggestion
        private static readonly IEnumerable<string> supportedTargetFrameworks = new string[] { TargetFramework.net8, TargetFramework.net7, TargetFramework.net6, TargetFramework.net48 };
        private static readonly IEnumerable<string> supportedInProcTargetFrameworks = new string[] { TargetFramework.net8, TargetFramework.net6 };
```

Reply:
Updated as suggested in the latest iteration pushed.